src/diffusers/models/attention.py

```diff
@@ -352,7 +358,11 @@ def gelu(self, gate):
         return F.gelu(gate.to(dtype=torch.float32)).to(dtype=gate.dtype)
 
     def forward(self, hidden_states, scale: float = 1.0):
-        hidden_states, gate = self.proj(hidden_states, scale).chunk(2, dim=-1)
+        hidden_states, gate = (
+            self.proj(hidden_states, scale).chunk(2, dim=-1)
+            if not USE_PEFT_BACKEND
+            else self.proj(hidden_states).chunk(2, dim=-1)
+        )
```

Comment:
A bit less repetition, WDYT?

```suggestion
        args = () if USE_PEFT_BACKEND else (scale,)
        hidden_states, gate = self.proj(hidden_states, *args).chunk(2, dim=-1)
```